turbinia/jobs/llm_artifacts_analyzer.py

```diff
@@ -0,0 +1,120 @@
+"""Job to execute LLM analysis on log, config and history files."""
+
+from __future__ import unicode_literals
+
+from turbinia import evidence as evidence_module
+from turbinia.jobs import interface
+from turbinia.jobs import manager
+from turbinia import workers
+from turbinia.workers.analysis import llm_analyzer as llm_analyzer_module
+
+LLM_ARTIFACTS = [
+    'BashShellConfigurationFile',
+    'BashShellHistoryFile',
+    'BashShellSessionFile',
+    'BourneShellHistoryFile',
+    'CShellConfigurationFile',
+    'FishShellConfigurationFile',
+    'FishShellHistoryFile',
+    'KornShellConfigurationFile',
+    'RootUserShellConfigs',
+    'RootUserShellHistory',
+    'ShellConfigurationFile',
+    'ShellHistoryFile',
+    'ShellLogoutFile',
+    'ShellProfileFile',
+    'TeeShellConfigurationFile',
+    'ZShellConfigurationFile',
+    'ZShellHistoryFile',
+    'JupyterConfigFile',
+    'NfsExportsFile',
+    'RedisConfigFile',
+    'DNSResolvConfFile',
+    'SambaConfigFile',
+    'SshdConfigFile',
+    'SshUserConfigFile',
+    'GKEDockerContainerLogs',
+    'DockerContainerConfig',
+    'NginxAccessLogs',
+    'ApacheAccessLogs',
+    'WordpressConfigFile',
+    'MicrosoftIISLogs',
+    'ElasticsearchAccessLog',
```

Comment:
no because the Analyzer itself chunk the data respecting the maximum input-token limit of the used model and make the model know that this is part 1 out of 5 of the config file, next prompt 1 out of 5 etc. The input limit for Gemini 1.0 pro is more than 30K tokens and each token is around 4 chrs, this issue will probably not exit when we upgrade to Gemini 1.5